capstone-app/src/app/app.module.ts

```diff
@@ -4,18 +4,18 @@ import { HistoryModule } from './modules/history/history.module';
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { AppComponent } from './app.component';
+import { HttpClientModule } from '@angular/common/http';
 
 @NgModule({
-  declarations: [
-    AppComponent
-  ],
+  declarations: [AppComponent],
   imports: [
     BrowserModule,
     HistoryModule,
     PhotosModule,
     MapsModule,
+    HttpClientModule,
   ],
   providers: [],
```

Comment:
Instead of having a getMockPhotos method in the component, add a provider here. It could be:

```
{
  provide: PhotoFetcher,
  useClass: MockPhotoFetcher,
}
```

Then you won't need to have two separate methods in the component, you can just remove the provider when you're ready to use the real thing.